inst/templates/bs4_book.html

```diff
@@ -122,6 +122,25 @@ <h2>On this page</h2>
     document.getElementsByTagName("head")[0].appendChild(script);
   })();
 </script>
+<!-- https://stackoverflow.com/a/49687594 -->
+<script type="text/x-mathjax-config">
+const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
+for (let popover of popovers){
+  const div = document.createElement('div');
+  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
+  div.innerHTML = popover.getAttribute('data-content');
+
+  // Will this work with TeX on its own line?
+  var has_math = div.querySelector("span.math");
+  if (has_math) {
+    document.body.appendChild(div);
+  	MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
+  	MathJax.Hub.Queue(function(){
+      popover.setAttribute('data-content', div.innerHTML);
```

Comment:
I think we can remove the child appended once the Math is generated.
```suggestion
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
```

It would mean not special div inserted in the DOM after new process by Mathjax